lib/client.js

```diff
@@ -378,6 +379,19 @@ class AssetComputeClient extends EventEmitter {
         }
         return response;
     }
+
+    async checkEventJornal(journalUrl) {
+        const ioEvents = new AdobeIOEvents({
+            accessToken: this.accessToken,
+            orgId: this.integration.technicalAccount.org
+        });
+        try {
+            await ioEvents.getEventsFromJournal(journalUrl);
```

Comment:
so just to be clear, the idea is:
- if the journal is set up, it will either return `200` with journal events, or `204` if there are no events yet.
- if the journal is not set up, it will return 500 internal server error

is this the fastest/ best api to check if the journal is set up? just wondering if there is a quicker api like validate journal or something

Reply:
@jdelbick that is correct.
Reviewing https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html#/Journaling ,  they seem to have only 2 apis. The other https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html#/Journaling/getValidateEventsFromJournal has the same response. So have checked on their slack channel. waiting to hear back.

Reply:
no other API which can do the validation